src/Console/ConfigurationResolver.php

```diff
@@ -350,6 +350,10 @@ private function parseRules()
         $rules = array();
 
         foreach (array_map('trim', explode(',', $this->options['rules'])) as $rule) {
+            if ('' === $rule) {
+                throw new InvalidConfigurationException('Empty rule name is not allowed.');
+            }
+
             if ('-' === $rule[0]) {
```

Comment:
It's still a bit confusing though. I think we should be throwing an invalid configuration exception if a dash occurs anywhere in the string apart from the first character.